lyon-amendment.md

```diff
@@ -14,9 +14,9 @@ The [Lancaster Consensus](lancaster-consensus.md) wrote:
 
 We have revised this to:
 
-No new release distribution in the Perl toolchain will specify a minimum perl
-prerequisite version (whether configure, build, runtime, or test) that has not
-been available for at least ten years.
+New release distributions in the Perl toolchain will support the last ten
```

Comment:
I believe the word "support" was deliberately avoided, because of the definition challenges involved

Reply:
What challenges?

Reply:
What do we commit ourselves to when using the word "support"

Reply:
That the toolchain works with every maintenance release of perl for the past ten years? Isn't the point of this amendment to tell the world which versions of perl the toolchain works with? Is there some concern that the toolchain will not work with recent versions?

Reply:
I favour the original text -- despite the double negative, it is the most accurate. I would just change "new release distribution" to "release of a distribution".

Reply:
Honestly I can’t even tell what the intention is - to disavow supporting anything older than 10 years, or to commit to supporting everything up to 10 years, or something else?

Also, what happens if Perl 5.x.y is 14 years old, but then 5.x.y+1 is suddenly released, does that bring it into support (I would assume not)?  It’s not accurate to consider releases as a linear set WRT time, but specifying minimum prereqs is a linear cut.  I’m not sure of the wisdom of putting strict date-based language on it when that’s not how it’s ultimately going to be implemented.

Reply:
Regarding the word “support” - would it be better to say “is tested against” or something like that?  Apologies if I’m coming in with too limited an understanding here.

Reply:
Here is my attempt at wording:

The toolchain group has decided to target a perl release that is at least 10 years old. By target we mean we will make an effort to insure the code works on perls going back 10 years. This is not a guarantee of support, but is an acknowledgement that we believe toolchain modules should be reliable on older version up to a reasonable point, and that point is perl that is 10 years old. Due to a Red Hat support cycle, we do make a one time exception extending this to 12 years as of 2023, 11 years in 2024, and finally 10 years moving forward from 2025 on.

Reply:
> Also, what happens if Perl 5.x.y is 14 years old, but then 5.x.y+1 is suddenly released, does that bring it into support (I would assume not)?

Perhaps it should be "major release" then?